test_standalone.py

```
@@ -0,0 +1,133 @@
+    wrapper._tool_description = func.__doc__ or ""
+    
+    return wrapper
+
+# Pinchwork tool
+@tool
+def pinchwork_delegate(task: str, skills_required: Optional[List[str]] = None, budget: float = 0.0) -> str:
+    """Delegate a task to the Pinchwork agent marketplace."""
+    try:
+        import httpx
+    except ImportError:
+        raise ImportError("httpx is required for Pinchwork integration. Install with: pip install httpx")
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)